src/main/java/nl/ordina/jobcrawler/service/DocumentService.java

```diff
@@ -0,0 +1,15 @@
+package nl.ordina.jobcrawler.service;
+
+import nl.ordina.jobcrawler.model.Vacancy;
+import nl.ordina.jobcrawler.scrapers.VacancyScraper;
+import org.jsoup.nodes.Document;
+
+public class DocumentService {
+
+    VacancyScraper vs;
+
+    public Document getDocument(String aSearchURL) {
+        return vs.getDocument(aSearchURL);
+    }
+
+}
```

Comment:
From what I understand, this class forwards the call to the static method in the abstract VacancyScraper. But why not calling the method directly from the VacancyScraper?

Reply:
we spoke about this in the scrum meeting and I showed you why this is not possible.